docs/development/triage.rst

```diff
@@ -0,0 +1,229 @@
+.. _triaging:
+
+Triaging Issues and Pull Requests
+=================================
+
+.. note::
+
+    This document serves mainly as a reference for the NAPALM maintainers,
+    but the users are equally welcome to read this document and understand our
+    process, and eventually suggest improvements.
+
+We triage Issues and Pull Requests (PR) using GitHub and ZenHub features:
+
+- :ref:`triage-labels`
+- :ref:`triage-milestone`
+- :ref:`triage-pipeline`
+
+.. _triage-labels:
+
+Labels
+++++++
+
+.. _triage-driver-labels:
+
+Driver labels
+-------------
+
+Each platform supported by NAPALM has associated a label, e.g., ``junos``, ``eos``,
+``ios``, ``iosxr``, ``vyos``, etc. It is mandatory that the maintainer to apply
+one or more of these labels.
+
+.. _triage-api-change-label:
+
+``api change``
+--------------
+
+If the Issue would imply a change in the API, or the PR introduces changes in
+the API. By API change we refer to changes in the getters output structure,
+methods signature, or any core changes that must be uniformly introduced across
+all the drivers.
+
+.. _triage-awesome-label:
+
+``awesome``
+-----------
+
+When someone adds or proposes something really awesome.
+
+.. _triage-blocked-label:
+
+``blocked``
+-----------
+
+Added in case we block the PR temporarly, or an Issue is currently blocked by
+other internal or external factors (PRs pending to be merged, other bugs to be
+solved a priori, etc.)
+
+.. _triage-bug-label:
+
+``bug``
+-------
+
+Whenever the behaviour reported in the Issue is different than it should, or the
+PR kills a bug.
+
+.. _triage-cannot-reproduce:
+
+``cannot reproduce``
+--------------------
+
+This refers to Issues only, and it is added when the maintainer(s) cannot
+reproduce the behaviour reported.
+
+.. _triage-deprecation-label:
+
+``deprecation``
+---------------
+
+Added only to PRs, when a API deprecation is introduced.
+
+.. _triage-documentation-label:
+
+``documentation``
+-----------------
+
+Can be added to both Issues and PRs, anything related to the documentation.
+
+.. _triage-duplicate-label:
+
+``duplicate``
+-------------
+
+Applicable to Issues only, to be added before closing a duplicate.
+
+.. _triage-epic-label:
+
+``epic``
```

Comment:
We should probably eliminate this. We can use the github projects instead if needed but zenhub is probably dead now we don't have multiple repos.